prac_05/emails.py

```diff
@@ -0,0 +1,39 @@
+def main():
+    email_to_name = {}
+    user_email = input("Enter Email :")
+    while user_email != '':
+        user_name = get_user_name(user_email)
+        is_name = input('Is {} your name? (Y/n) :'.format(user_name)).lower()
+        while is_name != 'y' and is_name != 'n':
+            print('Error: Invalid Input')
+            is_name = input('Is {} your name? (Y/n) :'.format(user_name)).lower()
+        if is_name == 'y':
+            email_to_name[user_name] = user_email
+        else:
+            user_name = input('Enter Name :')
+            email_to_name[user_name] = user_email
```

Comment:
this is the part i mentioned previously.
imo this can be extracted to make into another func

Reply:
Done, thanks